### ISSUE TITLE
Incorrect accessible name for the search modal

### ISSUE THREAD

**Issue by cyberbaloo** [suitenumerique/docs#2012]
## Bug Report

**Problematic behavior**
When the search modal is opened, it is announced as "share modal"

**Expected behavior/code**

The accessible name of the modal must vary according to its function. Here it is not a share modal but a search modal.

`<div class="ReactModal__Content ReactModal__Content--after-open c__modal c__modal--large" tabindex="-1" role="dialog" aria-modal="true">`

**Steps to Reproduce**
1. Open docs
2. Click on the "search" button using the keyboard
3. And then the bug happens!

**Possible Solution**

Add an `aria-labelledby="doc-search-modal-title"` attribute

This will announce the modal title which is not visible

`<h1 id="doc-search-modal-title" class="sc-5b83f41b-0 sc-458c56ea-0 hLzqHY dUDhBq sr-only">Rechercher des docs</h1>`

**Critère RGAA** : [Critère 7.1 : Chaque script est-il, si nécessaire, compatible avec les technologies d'assistance ?](https://accessibilite.numerique.gouv.fr/methode/criteres-et-tests/#7.1)

**Impact** 

A user who uses a screen reader is well informed that a modal has opened, but its title is not automatically announced when it opens. They will have to search to understand the context.

**Comment by FocusMode-coder** [comment]
Hey! I may be able to help with this bug.

I build custom fixes for Python projects and automation systems.

If you'd like help resolving this issue quickly feel free to reach out.

Happy to take a look 👍

**Comment by Ovgodd** [comment]
Hello @cyberbaloo ! This has been fixed on Crowdin (where translations are managed). It will be available in the next release.
